src/xdocs/config_imports.xml

```diff
@@ -1436,11 +1436,21 @@ import java.util.stream.IntStream;
             <td>sortStaticImportsAlphabetically</td>
             <td>whether <b>static imports</b> grouped by <b>top</b> or <b>bottom</b> option
                 are sorted alphabetically or not.
-                Attention: It is applied to all static imports as one group.</td>
+                Attention: It is applied to all static imports as one group
+                when the value of separatedStaticGroups is false.</td>
             <td><a href="property_types.html#boolean">Boolean</a></td>
             <td>false</td>
             <td>6.5</td>
           </tr>
+          <tr>
+            <td>separatedStaticGroups</td>
+              <td>whether <b>static imports</b> grouped by <b>top</b> or <b>bottom</b> option
+                    should be separated by, at least, one blank line or
+                  comment and aren't separated internally.</td>
```

Comment:
if we say "groups" ... we have to separate internally. Eventually as we get `groupsStatic` property we will allow separation inside static imports.